### ISSUE TITLE
Moderation UI - keep sorting after assigning on "assign" page

### ISSUE THREAD

**Issue by qubodup** [MTG/freesound#1519]
https://freesound.org/tickets/moderation/ would be user-friendlier if it kept the sorting method after assigning a user's sounds

To reproduce:
1. go to https://freesound.org/tickets/moderation/?order=new_count
2. press Assign to me on any user

What I expect:
I'm directed to https://freesound.org/tickets/moderation/?order=new_count

What actually happens:
I'm directed to https://freesound.org/tickets/moderation/

**Comment by ffont** [comment]
Good catch iwan, this is a bug. I'll look into it at some point. thanks!
